__test__/utils/query-builder.test.ts

```diff
@@ -0,0 +1,27 @@
+import * as QueryBuilder from "../../src/lib/utils";
+
+describe("QueryBuilder", () => {
+    describe("filterByTitle", () => {
+        const title = 'some title';
+        const filterQuery: string = QueryBuilder.filterByTitle(title);
+        const [key, value] = filterQuery.split("=");
+
+        test("should create query param key equal filter", () => {
+            expect(key).toEqual(QueryBuilder.QUERY_PARAM_NAME.FILTER);
+        });
+
+        describe("query value", () => {
+            test("should start with title", () => {
+                expect(value.startsWith(QueryBuilder.FILTER_PARAM.TITLE)).toBeTruthy();
+            });
+
+            test("should include eq", () => {
+                expect(value.includes("eq")).toBeTruthy();
+            });
+
+            test("should contains provided title", () => {
```

Comment:
A typo nitpick, I guess 😛 
```suggestion
            test("should contain provided title", () => {
```

Reply:
fixed